INSTALL.md

```diff
@@ -238,6 +238,19 @@ Deny from all
 
 For proper access to all pages of the website, enable the `mod_rewrite` module by issuing `a2enmod rewrite` on a terminal.
 
+If you are using Wamp, to run patient portal is needed change the following code:
```

Comment:
Say this .          If you are running the patient portal these items have to be configured.
Add it to the FAQ's also.

Reply:
Ok

Reply:
Under the FAQ say    I am getting a 500 error when trying to run portal dashboard. and then put the answer in that you use here